### PR TITLE
Tokenize text field in textFieldDidEndEditing

### PR DESCRIPTION
**Pull request by rizwan95**
The purpose of this PR is tokenize the text that is in the text field after the text field resigns its first responder status.

### CODE REVIEW

**Comment by rizwan95** [comment]
@ricardopereira Please review this PR

**Comment by ricardopereira** [comment]
@rizwan95 Thank you for the PR! Could you had a description please explaining the reason of this change?

So, the purpose of this PR is tokenize the text that is in the text field after the text field resigns its first responder status, right?

**Comment by rizwan95** [comment]
> @rizwan95 Thank you for the PR! Could you had a description please explaining the reason of this change?
> 
> So, the purpose of this PR is tokenize the text that is in the text field after the text field resigns its first responder status, right?

Yes, you are right. Sometimes the user might just type in a tag and not use the delimiter character and switch to another field. In that case, automatic tokenization in the `textFieldDidEndEditing` should be helpful.

**Comment by ricardopereira** [comment]
@rizwan95 Ok, to avoid breaking the existing apps with WSTagsField, I would suggest to create a new Boolean property called `shouldTokenizeAfterResigningFirstResponder` and only do that then it's true. By default, the property is `false`. Agree?

File: Source/WSTagsField.swift [hunk 1]
```diff
@@ -794,6 +794,9 @@ extension WSTagsField: UITextFieldDelegate {
     }
 
     public func textFieldDidEndEditing(_ textField: UITextField) {
+        if !isTextFieldEmpty{
```
Comment (ricardopereira) [inline Source/WSTagsField.swift:797]:
```suggestion
        if !isTextFieldEmpty {
```

This project doesn't have a linter but it should have one 😄 
Please add a space between the text and the curly brace. Thank you

**Comment by rizwan95** [comment]
> @rizwan95 Ok, to avoid breaking the existing apps with WSTagsField, I would suggest to create a new Boolean property called `shouldTokenizeAfterResigningFirstResponder` and only do that then it's true. By default, the property is `false`. Agree?

@ricardopereira : Yes, that sounds right. So shall we do that modification?

**Comment by ricardopereira** [comment]
> @ricardopereira : Yes, that sounds right. So shall we do that modification?

@rizwan95 Yes please.

File: Source/WSTagsField.swift (hunk 1 above)
Comment (rizwan95) [reply Source/WSTagsField.swift:797]: Done. Added it!

**Comment by rizwan95** [comment]
> See my comments. Thank you.

@ricardopereira Your suggestion has been integrated :)

**Comment by ricardopereira** [comment]
@rizwan95 I don't see them. Did you pushed your changes?

**Comment by rizwan95** [comment]
@ricardopereira  Please check now.

**Comment by rizwan95** [comment]
@ricardopereira that's awesome!

**Comment by leojkwan** [comment]
strange, pod install on 5.3.1 does not have this source.

**Comment by leojkwan** [comment]
Update: when I specify the specific master branch `, :git => 'https://github.com/whitesmith/WSTagsField.git' in my Podfile, the changes are pulled up, might be an issue with this projects's cocoapods trunk versioning?

**Comment by ricardopereira** [comment]
@leojkwan This hasn't been released yet. I'll try to make a new version this week. Sorry for the inconvenience.

**Comment by leojkwan** [comment]
Thanks @ricardopereira !

**Pull request merged**